Src/ILGPU/Runtime/MemoryBuffer.cs

```diff
@@ -786,28 +790,7 @@ public T[] GetAsArray(AcceleratorStream stream, TIndex offset, TIndex extent)
         /// <param name="stream">The used accelerator stream.</param>
         /// <returns>A new array holding the requested contents.</returns>
         public override byte[] GetAsRawArray(AcceleratorStream stream) =>
-            GetAsRawArray(stream, Index1.Zero, LengthInBytes).Array;
-
-        /// <summary>
-        /// Returns a sub view of the current view starting at the given offset.
-        /// </summary>
-        /// <param name="offset">The starting offset.</param>
-        /// <returns>The new sub view.</returns>
-        [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        [Obsolete("Use View.GetSubView(TIndex, Index) instead")]
-        public ArrayView<T, TIndex> GetSubView(TIndex offset) =>
-            View.GetSubView(offset);
-
-        /// <summary>
-        /// Returns a sub view of the current view starting at the given offset.
-        /// </summary>
-        /// <param name="offset">The starting offset.</param>
-        /// <param name="subViewExtent">The extent of the new sub view.</param>
-        /// <returns>The new sub view.</returns>
-        [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        [Obsolete("Use View.GetSubView(TIndex, Index) instead")]
-        public ArrayView<T, TIndex> GetSubView(TIndex offset, TIndex subViewExtent) =>
-            View.GetSubView(offset, subViewExtent);
+            GetAsRawArray(stream, Index1.Zero, (int)LengthInBytes).Array;
```

Comment:
MINOR: This calls into `GetAsRawArray(AcceleratorStream stream, long byteOffset, long byteExtent)`.

No need to cast `LengthInBytes` to `int`.

```suggestion
            GetAsRawArray(stream, Index1.Zero, LengthInBytes).Array;
```